Bonsai.Tinkerforge/BrickletAmbientLightV3.cs

```diff
@@ -3,15 +3,17 @@
 using System.Linq;
 using System.Reactive.Linq;
 using Tinkerforge;
+using System.Xml.Serialization;
 
 namespace Bonsai.Tinkerforge
 {
-    [DefaultProperty(nameof(Uid))]
+    [DefaultProperty(nameof(Device))]
     [Description("Measures ambient illuminance from an Ambient Light Bricklet 3.0.")]
     public class BrickletAmbientLightV3 : Combinator<IPConnection, long>
     {
-        [Description("The unique bricklet device UID.")]
-        public string Uid { get; set; }
+        [Description("Device data including address UID.")]
+        [TypeConverter(typeof(BrickletDeviceNameConverter))]
+        public TinkerforgeHelpers.DeviceData Device { get; set; }
```

Comment:
Looking back on this, I think the original idea of storing the full device data in the property was in case in later iterations of the module we want to do some downstream logic where we e.g. filter device streams based on their position in the stack or their connected modules. But I agree for now it is not necessary so will change to just store UID.

With regards to the type conversion, that is effectively how it works already, ConvertTo / ConvertFrom only queries the cached Dictionary<string, DeviceData>, the cache is updated from GetStandardValues calls, unless I've misunderstood the change here.